README.md

```diff
@@ -162,6 +162,61 @@ bundler: 'webpack' // Only visible when using webpack
 
 You can combine these options to precisely control when your component appears. For example, to show a component only when using Vite with HMR active, you would use both `when: 'hmr'` and `bundler: 'vite'`.
 
+## Provided Vite Configuration
+
+While the `apos` build (the code living in every module `ui/apos` directory) is fully preconfigured and doesn't allow for customization, the `public` build (the code imported within `ui/src/` ) is fully customizable and contains a minimal configuration to get you started:
+- A PostCSS plugin to handle core features as "Breakpoint Preview" (when enabled)
+- `Modules/` alias to simplify module within the same build 
+- `@/` alias to allow easy access to cross-module and cross-build source code
+
+### Pre-configured Aliases
+
+`Modules/` alias is available for both public and admin UI builds and allows you to import modules in your project without worrying about the relative path, but restricts you to only sources inside of `ui/src` directories.
+
+```javascript
+// Current file: modules/another-module/ui/src/index.js
+// Actual import path: modules/my-module/ui/src/lib/utils.js
+import utils from 'Modules/my-module/lib/utils.js';
+```
+
+`@/` alias is available for both public and admin UI builds and allows you to import files from the entire project source code.
+
+```javascript
+// Current file: any file in any module inside of the `ui/` folder
+// Actual path: modules/my-module/ui/src/lib/utils.js
+import utils from '@/modules/my-module/src/lib/utils.js';
+
+// Actual path: modules/my-module/ui/apos/mixins/SomeMixin.js
+import SomeMixin from '@/modules/my-module/apos/mixins/SomeMixin.js';
+```
+
+> Warning: You gain access to `public` builds from within the `apos` build, and vice versa, when using the `@/` alias. You should use it with caution, because it might lead to situations where imports are not resolved correctly. This would happen if the imported file (or its deep imports) contains `Modules/` aliased imports. In other hand `@/` is more developer friendly, allows auto-completion and is more intuitive and readable. Be sure to include mostly sources from your current build and ensure no imported sources contain `Modules/` aliased imports when cross-importing from another build.
```

Comment:
```suggestion
> Warning: You gain access to `public` builds from within the `apos` build, and vice versa, when using the `@/` alias. You should use it with caution, because it might lead to situations where imports are not resolved correctly. This would happen if the imported file (or its deep imports) contains `Modules/` aliased imports. On the other hand, `@/` is more developer friendly, allows auto-completion, and is more intuitive and readable. Be sure to include mostly sources from your current build and ensure no imported sources contain `Modules/` aliased imports when cross-importing from another build.
```